server/models/budgetModel.js

```diff
@@ -0,0 +1,30 @@
+const { DataTypes } = require("sequelize");
+const sequelize = require("../database/sequelize");
+
+const Budget = sequelize.define(
```

Comment:
extend from model

Reply:
removed sequelize.define and added class Model extends in commit 9a6b72d22fc7b1046356ec2c70ed77825b0bb0b4